tests/exporters/test_cds.py

```diff
@@ -0,0 +1,25 @@
+from src.exporters import CDSExporter
+
+
+class TestCDSExporter:
```

Comment:
I'm having trouble running these tests:
```
In [2]: !pytest tests
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --flake8
  inifile: /Users/tommylees/github/ml_drought/pytest.ini
  rootdir: /Users/tommylees/github/ml_drought
```

Have you seen that before? Do i need to install flake8?

Reply:
`conda install pytest-flake8 -c conda-forge` thanks @gabrieltseng !